### PR TITLE
Js webapp ending

### PR DESCRIPTION
**Pull request by jspaaks**
This PR adds some polishing of the text, fixes some final TODOs; I also found some inconsistencies between the code snippets in the README and the contents of the files, I corrected these.

~~Question: Should we consider replacing [the ``while`` loop](https://github.com/NLESC-JCER/run-cpp-on-web/blob/965eb7fa5f6391b86fa2959c86a5dc379ce53b85/js-webapp/newtonraphson.cpp#L11-L16) with a ``do-while`` loop? How we use the loop is the exact use case of while ``do-while`` exists.~~ I changed it to a ``do-while``.

### CODE REVIEW

File: js-webapp/README.md [hunk 1]
```diff
@@ -34,10 +36,11 @@ need to port it first**!
 _Newton (and his hair)._
 
 Now before you say _"That'll be so much slower than running it native!"_ or _"C/C++ from the browser? Impossible!"_,
-just hold your horses for a sec. With the right tools, it is possible to run C/C++ code in the browser, without any
-significant performance penalty. WebAssembly to the rescue, a low level language browsers can run and C++ can be compiled to. Using this approach, Gabriel Cuvillier was able to run the video
-game [Doom 3 in the browser](http://wasm.continuation-labs.com/d3demo/) and didn't find any performance problems. And if it works for video games, it will likely work
-for your research software, too.
+just hold your horses for a sec. With the right tools, it is possible to run C/C++ code in the browser, without a
+significant performance penalty. For example, Gabriel Cuvillier was able to run the video game _Doom 3_ [in the
```
Comment (sverhoeven) [inline js-webapp/README.md]: Taking #10 into account this is a bit to optimistic
```suggestion
just hold your horses for a sec. With the right tools, it is possible to run C/C++ code in the browser, with an acceptable performance penalty. For example, Gabriel Cuvillier was able to run the video game _Doom 3_ [in the
```

File: js-webapp/README.md [hunk 2]
```diff
@@ -164,7 +168,8 @@ Now we're ready to move on to the WebAssembly part.
 
 ### Binding
 
-The binding of the C++ code:
+To use the Newton-Raphson code from JavaScript, we'll need to define the _bindings_ file. The binding provides entry
+points to C++ methods and classes, which can then be used by JavaScript (after compilation). For our Newton-Raphson code, the binding file looks like this:
```
Comment (sverhoeven) [inline js-webapp/README.md]: It is hard to describe what binding does. Entry points are a bit vague to me.

Some alternatives
* The binding allows to map C++ constructs to their JavaScript equivalent
* The binding allows compiled code to be used in a natural way in JavaScript
* The binding allows compiled code to be called from JavaScript
* The binding provides method to connect and interact between JavaScript and compile C++

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Nice to see a bit of re-ordering can shorten sentences and make them easier to read.
See some inlined suggestions

File: js-webapp/README.md (hunk 2 above)
Comment (jspaaks) [reply js-webapp/README.md]: changed

**Pull request merged**